app/decorators/user_decorator.rb

```diff
@@ -49,22 +49,18 @@ def enriched_colors
     end
   end
 
-  def config_font_name
-    config_font.gsub("default", Settings::UserExperience.default_font)
-  end
-
   def config_body_class
     body_class = [
-      config_theme.tr("_", "-"),
-      "#{config_font_name.tr('_', '-')}-article-body",
+      setting.config_theme.tr("_", "-"),
```

Comment:
These changes are necessary to pull the customization values from `users_settings` and not `users`.

Reply:
@msarit This decorator also needs changes in the `enriched_colors` method. This currently uses `bg_color_hex` and `text_color_hex` which only work because of this block in `user.rb`:

```ruby
      # Getters and setters for mapped profile attributes
      Profile::MAPPED_ATTRIBUTES.each do |profile_attribute, user_attribute|
        define_method(user_attribute) { profile&.public_send(profile_attribute) }
        define_method("#{user_attribute}=") do |value|
          profile&.public_send("#{profile_attribute}=", value)
        end
      end
```

Instead, this should be using the `brand_color1` and `brand_color2 values from the user's settings.

Reply:
Done!